### PR TITLE
feat(Offer): integrate ProductSelector

### PR DESCRIPTION
**Pull request by guilhermespopolin**
## What?

- integrate `ProductSelector` into _offer_ page

## Why?

To enable _Pentagon House Cross Selling_

**Ticket(s): [GRW-1337](https://hedvig.atlassian.net/browse/GRW-1337)**

image

### CODE REVIEW

**Comment by guilhermespopolin** [comment]
Current dependencies on/for this PR:
* master
  * **PR #1121** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/web-onboarding/1121" target="_blank">Graphite</a> 
    * **PR #1122** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/web-onboarding/1122" target="_blank">Graphite</a> 
      * **PR #1123** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/web-onboarding/1123" target="_blank">Graphite</a> 
        * **PR #1124** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/web-onboarding/1124" target="_blank">Graphite</a> 
          * **PR #1125** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/web-onboarding/1125" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/web-onboarding/1125?utm_source=stack-comment).

File: src/client/pages/Offer/ProductSelector/Selector.tsx [hunk 1]
```diff
@@ -7,6 +7,8 @@ import { Product } from 'pages/Offer/types'
 import { StandaloneProductCard } from './StandaloneProductCard'
 import { AdditionalProductCard } from './AdditionalProductCard'
 
+const SEARCH_PARAM = 'type'
```
Comment (robinandeer) [inline src/client/pages/Offer/ProductSelector/Selector.tsx:10]: nice!

File: src/client/pages/Offer/ProductSelector/Selector.tsx [hunk 2]
```diff
@@ -124,7 +126,7 @@ export const Selector = ({
 
                     setSelectedProductsByCategory(newState)
 
-                    history.push({
+                    history.replace({
```
Comment (robinandeer) [inline src/client/pages/Offer/ProductSelector/Selector.tsx:129]: ahh I was thinking about this earlier -- great change

**Review by robinandeer — APPROVED** [review]
Amazing that you made it work with only this ONE feature toggle :D

Sure we are requesting too much data but okey to get this shipped

My only comment is that the padding in the main coverage cards looks little off (too much)

**Comment by robinandeer** [comment]
**Graphite Merge Job**

Current status: ⏳ Queued to merge

This pull request is currently queued to be merged as part of a stack.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/web-onboarding/1125?utm_source=stack-comment).


Job Reference: zxLtjZlK3VtfUNAuwuaI

**Pull request merged**